modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -6,12 +6,10 @@ module AppealsApi::HigherLevelReviews::V0
   class HigherLevelReviewsController < AppealsApi::V2::DecisionReviews::HigherLevelReviewsController
     include AppealsApi::OpenidAuth
 
-    FORM_NUMBER = '200996_WITH_SHARED_REFS'
```

Comment:
We've been a bit confused by `FORM_NUMBER` before - I've left it in controllers where it actually referred to a form number (10182, 200995, or 200996), but I've removed it everywhere else. 

In this specific case, this HLR controller now inherits the 200996 `FORM_NUMBER` from its parent controller, and the difference between decision reviews and segmented API schemas now comes the API name and version, rather than the `200996_WITH_SHARED_REFS` "form number"